src/sentry/static/sentry/app/components/fileIcon.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import {IconFile} from 'app/icons';
+import theme from 'app/utils/theme';
+
+const FILE_EXTENSION_TO_ICON = {
+  jsx: 'react',
+  tsx: 'react',
+  js: 'javascript',
```

Comment:
Should `ts` go to javascript too?

Reply:
Yeah, I mapped it. I would love a dedicated TS icon though 🙂